appserver/web/weld-integration/src/test/java/org/glassfish/cdi/transaction/TransactionScopedContextImplTest.java

```diff
@@ -1,290 +0,0 @@
-/*
- * Copyright (c) 2012, 2020 Oracle and/or its affiliates. All rights reserved.
- * Copyright (c) 2021 Contributors to the Eclipse Foundation
- *
- * This program and the accompanying materials are made available under the
- * terms of the Eclipse Public License v. 2.0, which is available at
- * http://www.eclipse.org/legal/epl-2.0.
- *
- * This Source Code may also be made available under the following Secondary
- * Licenses when the conditions for such availability set forth in the
- * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
- * version 2 with the GNU Classpath Exception, which is available at
- * https://www.gnu.org/software/classpath/license.html.
- *
- * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
- */
-
-package org.glassfish.cdi.transaction;
-
-import javax.naming.Context;
-import javax.naming.InitialContext;
-import javax.naming.NamingException;
-
-import org.easymock.EasyMockSupport;
-import org.junit.jupiter.api.AfterEach;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
-
-import jakarta.enterprise.context.ContextNotActiveException;
-import jakarta.enterprise.context.spi.Contextual;
-import jakarta.enterprise.context.spi.CreationalContext;
-import jakarta.enterprise.inject.spi.PassivationCapable;
-import jakarta.transaction.Status;
-import jakarta.transaction.TransactionScoped;
-import jakarta.transaction.TransactionSynchronizationRegistry;
-
-import static org.easymock.EasyMock.expect;
-import static org.easymock.EasyMock.isA;
-import static org.easymock.EasyMock.same;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertSame;
-import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-
-/**
- * @author <a href="mailto:j.j.snyder@oracle.com">JJ Snyder</a>
- */
-public class TransactionScopedContextImplTest {
```

Comment:
Why are you deleting unit tests? It would be better to update it and enhance it instead. This is not how you should update dependencies (EasyMock) ;)

Reply:
It's an overly mocked test, that mocks so much it's a question what it actually tests still. I don't think it's possible to bring it back without downgrading EasyMock, as newer versions of EasyMock aren't capable anymore of mocking InitialContext.